lib/logstash/outputs/zeromq.rb

```diff
@@ -107,7 +107,7 @@ def server?
 
   public
   def receive(event)
-    return unless output?(event)
+    
```

Comment:
yeah this is a current limitation of the ruby scripts, this aint sed. :(
One way to do it would be to split files and compare each lines and that would be a bit slower.